crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -9401,10 +9401,10 @@ Ihr spürt die alte Wut Eurer Axt.
 %s fällt aus der Luft.
 %%%%
 The tentacle is hauled back through the portal!
-Der Tentakel wird durch das Portal zurückgeschleppt!
+Das Tentakel wird durch das Portal zurückgeschleppt!
```

Comment:
This should be:
Das Tentakel wird in das Portal zurückgerissen!

Reply:
done